npm/npm_import.bzl

```diff
@@ -341,14 +363,9 @@ def npm_translate_lock(
 
         pnpm_version: pnpm version to use when generating the @pnpm repository. Set to None to not create this repository.
 
-        data: Data files required by this repository rule.
-
-            When any of these files changes it causes the repository rule to re-run.
+        package_json: Deprecated.
```

Comment:
Lets create an issue for 2.0 or some type of `TODO(2.0)` comment so we don't forget this?

Reply:
✅  left a `TODO(2.0)` which is what I've been using throughout